lib/expand.js

```diff
@@ -8,8 +8,8 @@ function setLogger(bunyanLogger) {
   rootLogger = logger = bunyanLogger;
 }
 
-function expand(specifications, ...exporters) {
-  const expander = new Expander(specifications, ...exporters);
+function expand(specifications, configuration, ...exporters) {
```

Comment:
I think there are other modules (e.g., `shr-es6-export`, `shr-json-schema-export`) that also use `shr-expand` -- and likely need to be updated to support this signature change.  I think it's mainly in their tests.

Reply:
You are correct, this broke the changes in `shr-es6-export`. I added a PR here https://github.com/standardhealth/shr-es6-export/pull/61 to fix that.